backend/Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM node:18-alpine as development  
 
 # Set the working directory inside the container
-WORKDIR /backend
+WORKDIR /app
```

Comment:
I have a doubt:
Why is the working directory set to /app when this is the dockerfile for the backend and commands below are executed relative to the backend?

Reply:
Workdir mean where docker will put the content inside his Virtual Machine
it's not related to the current directory. 

I often use /app so people remember that the WORKDIR is inside docker and not related to the project directory structure

Reply:
oh right. but don't you feel that it's confusing since our frontend codebase lies in /app?

Reply:
Yes i agree. 
I don't find a better name to descript it. 

What do you propose ?

Reply:
How about /docker_workdir or /docker_context or /docker_app ?

Reply:
I think /docker_workdir is good